pkg/agent/cli/agent.go

```diff
@@ -15,19 +15,19 @@ import (
 )
 
 type Agent struct {
-	cfg            *config.Config
+	cfg            *config.Agent
 	cs             *csock.CSock
 	activeProfiles map[int]*agent.ProfileSession
 	id             id.ID
 	u              upstream.Upstream
 }
 
-func New(cfg *config.Config) *Agent {
+func New(cfg *config.Agent) *Agent {
```

Comment:
@petethepig Seems that this is unused, is that possible?

Reply:
Yeah, I think we'll delete it soon